copulas/multivariate/gaussian.py

```diff
@@ -160,16 +165,21 @@ def sample(self, num_rows=1):
         return pd.DataFrame(data=res)
 
     def to_dict(self):
-        distributions = {
+        distribs = {
             name: distribution.to_dict() for name, distribution in self.distribs.items()
         }
+        distribution = self.distribution
+        if isinstance(self.distribution, dict):
```

Comment:
This allows us to save instances of this model where the univariate marginals are manually specified. On a related note, we need to check whether the Univariate instance can be saved/loaded while preserving the Parametric/Bounded flags.